app.json

```
@@ -0,0 +1,43 @@
+{
+  "expo": {
+    "name": "flat-finder",
+    "slug": "flat-finder",
```

```suggestion
    "name": "FlatFinder",
    "slug": "FlatFinder",
```
Maybe this is better?